test/e2e/volumes.go

```diff
@@ -839,6 +840,64 @@ var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
 	})
 
 	////////////////////////////////////////////////////////////////////////
+	// vSphere
+	////////////////////////////////////////////////////////////////////////
+
```

Comment:
we should not change file name for existing e2e tests. for new e2e tests we can keep meaningful name.
followed the same pattern as other Its (test specs) in the file.

Test Steps:

1. Create Volume (VMDK)
2. Create a POD (POD-A) using busybox image, with volume path set to path of the VMDK created in Step 1.
3. Create a file index.html with the random content on the mounted volume.
4. Create POD Spec using Volume Path for the VMDK created in Step 1. Also set "1234" as FSGroup in the POD Spec.
5. Create POD (POD-B) using the pod spec created in the step 4.
6. Verify FS Group is set correctly on the mounted volume.
7. Verify file created from POD-A is accessible and content is matching.